packages/rest/src/coercion/validator.ts

```diff
@@ -65,9 +65,19 @@ export class Validator {
   isAbsent(value: any) {
     if (value === '' || value === undefined) return true;
 
+    const spec: ParameterObject = this.ctx.parameterSpec;
```

Comment:
Maybe refactor the code here to be:

```ts
const spec: ParameterObject = this.ctx.parameterSpec;
const valueIsNull = value === 'null' || value === null;

if (spec.schema) {
  if (schema.type === 'object' && valueIsNull) return true;
} else {
  if (spec.in === 'query' && spec.content && valueIsNull)
    return true;
}
```

if I understand the logic here correctly.

Reply:
@jannyHou that is perfect, I will change as per your suggestion.

Reply:
changes done as suggested